packages/vm/examples/module_size.rs

```diff
@@ -0,0 +1,62 @@
+use std::fs::File;
+use std::io::Read;
+use std::mem;
+
+use clap::{App, Arg};
+
+use cosmwasm_vm::internals::compile_and_use;
+use cosmwasm_vm::internals::make_runtime_store;
+use cosmwasm_vm::Size;
+use wasmer::Module;
+
+pub fn main() {
+    let matches = App::new("Module size estimation")
+        .version("0.0.1")
+        .author("Mauro Lacy <mauro@lacy.com.es>")
+        .arg(
+            Arg::with_name("WASM")
+                .help("Wasm file to read and compile")
+                .required(true)
+                .index(1),
+        )
+        .get_matches();
+
+    // File
+    let path = matches.value_of("WASM").expect("Error parsing file name");
+    let mut file = File::open(path).unwrap();
+    mem::drop(matches);
+
+    // Read wasm
+    let mut wasm = Vec::<u8>::new();
+    file.read_to_end(&mut wasm).unwrap();
+    mem::drop(file);
+
+    // Report size
+    let wasm_size = wasm.len();
+    println!("wasm size: {} bytes", wasm_size);
+
+    // Compile module
+    let memory_limit = Some(Size::mebi(10));
+    let module = compile_and_use(&wasm, memory_limit).unwrap();
+    mem::drop(wasm);
+
+    let serialized = module.serialize().unwrap();
+    mem::drop(module);
+
+    // Deserialize using make_runtime_store()
+    let store = make_runtime_store(memory_limit);
+
+    let module = unsafe { Module::deserialize(&store, &serialized) }.unwrap();
```

Comment:
What about wrapping those two in a named function which we later can search? Then we get the allocations for
1. Creating the headless store
2. Creating the Module using this store
as one report. This is what we need at high level: memory consumtion of Module including store.

Reply:
Sure, but, you mean, in the vm API, or in `module_size`?

Reply:
I mean here in this example. Just pull out a function that grups the two. If Rust does some inlining we don't want because it is the same file, we can probably use `#[inline(never)]` or something like that.

Reply:
`#[inline(never)]` was in fact needed. Thanks for that.